packages/core/src/api/app/FeatureFlags.test.tsx

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FeatureFlags as FeatureFlagsImpl } from './FeatureFlags';
+import { FeatureFlagState } from '../apis/definitions/featureFlags';
+
+describe('FeatureFlags', () => {
+  beforeEach(() => {
+    window.localStorage.clear();
+  });
+
+  describe('#getFlags', () => {
+    let FeatureFlags;
+
+    beforeEach(() => {
+      FeatureFlags = new FeatureFlagsImpl();
```

Comment:
Typically only types, enums and constructors are PascalCase in JS/TS, this should be camelCase

Reply:
Fixed in c9c255c84732af7e2f9ca6e31ccd98297487947d